test_bond/test/exercise_bond.py

```diff
@@ -148,6 +148,16 @@ def test_die_ignore_death(self):
         self.assertFalse(bond.wait_until_broken(rospy.Duration(1.0)))
         self.assertTrue(bond.wait_until_broken(rospy.Duration(10.0)))
 
+    # Shutdown() before start()
+    def test_shutdown_before_start(self):
+        self.bond = bondpy.Bond(TOPIC, id)
+
+        try:
```

Comment:
This `try` shouldn't be necessary. The test runner already wraps each test and will fail if there's an uncaught exception.

Reply:
Okay.